contrib/dnscrypt-proxy/files/dnscrypt-proxy

```diff
@@ -0,0 +1,10 @@
+# dnscrypt-proxy service
+
+type            = process
+command         = /usr/bin/dnscrypt-proxy -config /etc/dnscrypt-proxy/dnscrypt-proxy.toml
+logfile         = /var/log/dnscrypt-proxy/dnscrypt-proxy.log
```

Comment:
is there any reason to put the log in a subdir?

Reply:
dnscrypt-proxy has separate-file logging for handled DNS queries, suspicious domains, etc. Upstream (and other distros) group them together in a dnscrypt-proxy folder as per the default config.

Reply:
ah, understood